roles/storage-cinder/tasks/deprovision.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Delete Service Account
+  command: kubectl delete serviceaccount {{service_account}} -n {{namespace}}
+  when: sa.stdout == ""
+
+- name: Revoke anyuid access to {{service_account}} SA if using OCP
```

Comment:
If you delete the SA doesn't this go away with it?

Reply:
That is correct. Will prune it.